domains/_vercel.q.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "syedtahseen",
+    "email": "itxtahseen@gmail.com"
+  },
+  "record": {
+    "TXT": ["xproject-xi.vercel.app"]
```

Comment:
This should be a vercel verification code.

Reply:
Unresolved as it's still just the domain name and not an actual verification code.